From 2b672a2928d57b1132644ee4da1d6c664050bce1 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sun, 8 Jul 2012 22:20:15 +0200
Subject: [PATCH] fix sox detection with sox >= 14.4.0

sox 1.14.0 changed the string that is printed out on --version, breaking the simply string matching done;
add a new case to cover also this new version

BUG: 301544

(cherry picked from commit 79cd49cac3a6b7031556aae53ce3ecff8c360cb9)
---
 plugins/encoder/sox/k3bsoxencoder.cpp |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/plugins/encoder/sox/k3bsoxencoder.cpp b/plugins/encoder/sox/k3bsoxencoder.cpp
index 62d2bcd..72a3dc0 100644
--- a/plugins/encoder/sox/k3bsoxencoder.cpp
+++ b/plugins/encoder/sox/k3bsoxencoder.cpp
@@ -69,6 +69,9 @@ public:
             if ( pos >= 0 ) {
                 pos += 17;
             }
+            else if ( ( pos = out.indexOf( "sox:      SoX v" ) ) >= 0 ) {
+                pos += 15;
+            }
             else if ( ( pos = out.indexOf( "sox: SoX v" ) ) >= 0 ) {
                 pos += 10;
             }
-- 
1.7.10.4

